feat (jkube-kit/resource) : Add toggle to preserve HelmParameter case (#2369)
eclipse/jkube: jkube-kit/doc/src/main/asciidoc/inc/helm/_jkube_helm.adoc

  The parameters can also represent a Golang expression
|

| preserveParameterCase

> | A boolean toggling whether the parameter case is preserved when generating the Values.yaml file, off by default

Reviewer: nit, would be better to write default value just like other fields.
```suggestion
| A boolean toggling whether the parameter case is preserved when generating the Values.yaml file

Defaults to `false`
```

Author: Done
